src/main/resources/liquibase/changelog.xml

```diff
@@ -122,8 +122,7 @@
     <!-- <include file="liquibase/202308211518-rename-rhel-product-ids.xml"/> -->
     <include file="liquibase/202307261851-event-change-event-type-data.xml"/>
     <include file="liquibase/202308210806-add-metric-id-instance-measurements.xml"/>
-    <!-- Uncomment once the above changeset has landed in prod -->
-    <!-- <include file="liquibase/202308210906-drop-uom-column-instance-measurements.xml"/> -->
+    <include file="liquibase/202308210906-drop-uom-column-instance-measurements.xml"/>
     <include file="liquibase/202308231356-add-metric-id-tally-measurements.xml"/>
     <!-- Uncomment once the above changeset has landed in prod -->
     <!-- <include file="liquibase/202308231456-drop-uom-column-tally-measurements.xml"/> -->
```

Comment:
Do we have card for dropping the column off `tally_measurements` as well?  I did a search on the backlog but didn't see anything.  If not, should we roll dropping `uom` on `tally_measurements` into this?

Reply:
Yes, I created this ticket https://issues.redhat.com/browse/SWATCH-1657 for dropping `uom` from `tally_measurements` which we can't do yet because https://issues.redhat.com/browse/SWATCH-430 is not released yet.